tests/test_loaders_log.py

```diff
@@ -9,20 +10,21 @@
 
 
 @pytest.mark.parametrize(
-    "input_file, expected_mapping",
+    "path, uri, input_file, expected_mapping",
     [
-        ("/dir/test.evtx", "/sysvol/windows/system32/winevt/logs/test.evtx"),
-        ("/dir/test.evt", "/sysvol/windows/system32/config/test.evt"),
+        ("/dir/*.evt*", None, "/dir/test.evtx", "/sysvol/windows/system32/winevt/logs/test.evtx"),
+        ("/dir/*.evt*", None, "/dir/test.evt", "/sysvol/windows/system32/config/test.evt"),
+        ("/source/iis.log", "log:///dir/with/files/*.log?hint=iis", "/source/iis.log", "/sysvol/files/logs/iis.log"),
```

Comment:
I do like this "hint" parameter! It makes it possible to specify the type of *.log file (or whatever standard log output format) that should be interpreted for each file separately. 

We need to document this clearly somewhere, such that it is option is generally known